<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTimeDlg</class>
 <widget class="QWidget" name="SetTimeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/navigationbar/设置.png</normaloff>:/navigationbar/设置.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="layoutDigitHour"/>
  </widget>
  <widget class="QPushButton" name="pushButtonHourInc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHourDec">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinuteInc">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinuteDec">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="layoutDigitMinute"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonHourInc</tabstop>
  <tabstop>pushButtonHourDec</tabstop>
  <tabstop>pushButtonMinuteInc</tabstop>
  <tabstop>pushButtonMinuteDec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
